charts/kube-ovn/templates/kube-ovn-crd.yaml

```diff
@@ -503,6 +503,15 @@ spec:
                     type: string
                 qosPolicy:
                   type: string
+                bgpSpeaker:
+                  type: object
+                  properties:
+                    enabled:
+                      type: boolean
+                    parameters:
+                      type: array
+                      items:
+                        type: string
```

Comment:
How about split the `parameters` property into multiple fields for easier use? We can refer to [BGPPeer configuration](https://github.com/metallb/metallb/blob/a207a6e5a26dfa10d8552f099d3eb93eaa095fec/api/v1beta2/bgppeer_types.go#L25) used by MetalLB.

Reply:
Sure, I'll extract the parameters the speaker already supports from the metallb config

Reply:
done, i still left the extraArgs just in case